ebmbot/slack.py

```diff
@@ -46,10 +51,27 @@ def notify_slack(
             sleep(1)
             error = err
 
-    logger.error(
-        "Could not notify slack",
-        channel=channel,
-        message=message_text,
-        thread_ts=thread_ts,
-        error=error,
+    msg_kwargs["text"] = COULD_NOT_NOTIFY_SLACK
+    msg_kwargs["blocks"] = get_slack_error_blocks(message_text, error)
+    try:
+        slack_client.chat_postMessage(**msg_kwargs)
+    except Exception:
```

Comment:
Suggestion: just `pass` here and add a `finally:` block to always log the error?  If we get here, we know there was an error of some sort, so even if we can report it back to slack, it'd be useful to have it in the app's logs